include/boost/beast/websocket/stream_fwd.hpp

```diff
@@ -12,18 +12,18 @@
 
 #include <boost/beast/core/detail/config.hpp>
 
+#ifndef BOOST_BEAST_DOXYGEN
+
 //[code_websocket_1h
 
 namespace boost {
 namespace beast {
 namespace websocket {
 
-#ifndef BOOST_BEAST_DOXYGEN
```

Comment:
Why did you remove the ifdef? This will probably confuse doxygen

Reply:
I didn't remove it. I just moved it outside the quickbook comment to fix this small bug in the docs that were rendering `BOOST_BEAST_DOXYGEN` sometimes.

![image](https://user-images.githubusercontent.com/5369819/177435458-2eea70c2-d172-449e-af8c-d9f1d6bd9b74.png)

To be honest, I just checked https://www.boost.org/doc/libs/1_79_0/libs/beast/doc/html/beast/using_websocket.html and it seems like this only happens locally. I have no idea why.